python/examples/advanced/Log_Rotation_for_Streaming_Data/Streaming_Data_with_Log_Rotation.ipynb

```diff
@@ -0,0 +1,1221 @@
+       "      <td>THB</td>\n",
+       "      <td>TRY</td>\n",
+       "      <td>TWD</td>\n",
+       "      <td>USD</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>5 rows × 28 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "               ARS       AUD        BRL       CAD       CHF          CLP  \\\n",
+       "15m     4629222.76  29167.28  105202.49  26189.14  19424.96  18163033.49   \n",
+       "last    4629222.76  29167.28  105202.49  26189.14  19424.96  18163033.49   \n",
+       "buy     4629222.76  29167.28  105202.49  26189.14  19424.96  18163033.49   \n",
+       "sell    4629222.76  29167.28  105202.49  26189.14  19424.96  18163033.49   \n",
+       "symbol         ARS       AUD        BRL       CAD       CHF          CLP   \n",
+       "\n",
+       "              CNY        CZK        DKK       EUR  ...       NZD       PLN  \\\n",
+       "15m     146442.74  481208.62  155741.55  19170.13  ...  32247.65  89876.17   \n",
+       "last    146442.74  481208.62  155741.55  19170.13  ...  32247.65  89876.17   \n",
+       "buy     146442.74  481208.62  155741.55  19170.13  ...  32247.65  89876.17   \n",
+       "sell    146442.74  481208.62  155741.55  19170.13  ...  32247.65  89876.17   \n",
+       "symbol        CNY        CZK        DKK       EUR  ...       NZD       PLN   \n",
+       "\n",
+       "             RON        RUB        SEK       SGD        THB        TRY  \\\n",
+       "15m     72010.47  1115075.7  204322.49  28201.47  715312.44  354510.87   \n",
+       "last    72010.47  1115075.7  204322.49  28201.47  715312.44  354510.87   \n",
+       "buy     72010.47  1115075.7  204322.49  28201.47  715312.44  354510.87   \n",
+       "sell    72010.47  1115075.7  204322.49  28201.47  715312.44  354510.87   \n",
+       "symbol       RON        RUB        SEK       SGD        THB        TRY   \n",
+       "\n",
+       "               TWD       USD  \n",
+       "15m     26353962.8  20270.23  \n",
+       "last    26353962.8  20270.23  \n",
+       "buy     26353962.8  20270.23  \n",
+       "sell    26353962.8  20270.23  \n",
+       "symbol         TWD       USD  \n",
+       "\n",
+       "[5 rows x 28 columns]"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "example_path = os.path.join(\"mock_input\", \"moc_1655923486.322933.json\")\n",
+    "example_df = pd.read_json(example_path)\n",
+    "example_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "##  Example\n",
+    "This example will be more like what you'll see in an environment. Imagine we want to see our data **every day** you'll want to have at logs **every thirty**, but you don't want to roll the logger over with only one data point, so we'd work with every 5 hours for instance.\n",
```

Comment:
Both paragraphs here are really confusing. Every thirty what? the rolling logger is every 5 hours or every hour? Either way, the code below uses 15 minutes as rolling interval, so it should be changed as well

Reply:
We should also explain why is there a timer with random seconds, making it clear that is to reflect a production environment, where data arrives at random intervals.

Reply:
Absolutely thanks! Will rework